web/src/components/about/FAQ.vue

```diff
@@ -6,11 +6,23 @@
     <br>
     <div
       v-for="q in questions"
-      :key="q.question"
+      :key="q.question()"
```

Comment:
This resolves to a string, rather than a function, which makes a vue error go away.